english/light-fire/stop-drop-roll/start-licking-yourself/start-licking-yourself.md

```diff
@@ -1,2 +1,14 @@
 
 You are covered in thick marshmallow and stuck to the floor. You make an attempt to lick-your-way-out.
+
+Out of the goop emerges a marshmellow dog. The dog apparently has better licking skills than you. She licks you clean in no time.
+
+Free from the floor at last, you rise to your feet and gaze appreciatively at the dog. You decide it's probably best not to risk petting her, seeing as she's made out of marshmellow and all. 
+
+You decide you want to name her. Do you call her:
+
+[Alba](./alba/alba.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.